### DISCUSSION TITLE
Soon? What is soon....

### DISCUSSION THREAD

**Discussion by ghost** [supabase/supabase#1295] [General]
I would love to know when Cloud functions are expected? Is soon next month, or end of this year?

@kiwicopple  Any news available to share with us?

**Comment by ghost** [comment]
@kiwicopple I'm still very curious on this, is there any news available?

Especially since all the reactions from the community on https://aws.amazon.com/about-aws/whats-new/2021/04/amazon-rds-postgresql-integrates-aws-lambda/

**Comment by quicksnap** [comment]
Also looking forward to this. 

FWIW, I'm currently using Vercel for easy-mode AWS Lamda and simple FE deployments. Would love to keep things in Supabase, though, if and when possible.

**Reply by cdedreuille** [reply]
From a comment on their latest Youtube video (https://www.youtube.com/watch?v=uWJmUTCFdak&t=1s) :
"Correct. Functions will also solve this. We will ship our Workflows engine first (https://supabase.io/blog/2021/04/02/supabase-workflows) which should be ready around the end of July. This will allow you to call functions (including other services outside of Supabase) and then we work on providing Functions ourselves too.."

**Comment by srvmux** [comment]
for me I'm looking forward to having storage on the self hosted version, and idk if it's even a priority for them... but in the mean time I'm heaving fun trying to learn the rest of supabase

**Reply by inian** [reply]
This is a good starting point @oussamabouchareb - https://github.com/supabase/supabase/pull/1007
This isn't completely "local" since it still requires access to a S3 bucket. I am trying to figure out if there is a way around that.

**Reply by srvmux** [reply]
> This is a good starting point @oussamabouchareb - #1007
> This isn't completely "local" since it still requires access to a S3 bucket. I am trying to figure out if there is a way around that.

when I was browsing minio the other day it said that they were S3 compatible, wouldn't that work with supabase?
